CHANGELOG.md

```diff
@@ -84,7 +84,7 @@ corresponding [documentation](/docs/migration/Version_0.1.x_0.3.x.md).
 - Bump alpine (#749)
 - Bump alpine (#750)
 - Bump alpine (#752)
-- Bump alpine in /edc-controlplane/edc-controlplane-memory/src/main/docker (#753)
+- Bump alpine in /edc-controlplane/edc-runtime-memory/src/main/docker (#753)
```

Comment:
yes, but that would be incorrect, because it is not just a controlplane, but an entire runtime (including the data plane). also, it contains modified contents, i.e. no Azure Vault anymore.